codegen/Dockerfile

```diff
@@ -26,5 +26,11 @@ ENV PATH="${PATH}:/usr/local/go/bin:/root/go/bin"
 RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31
 RUN go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3
 
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable --profile minimal; \
+    . /root/.cargo/env; \
```

Comment:
```suggestion
    . $HOME/.cargo/env; \
```

can we use this ^ instead? I was puzzled by the `/root` for a second.

Reply:
I think we can. I will try `$HOME` today.

Reply:
I did some simple tests, and I can change this line. But `Env` needs an explicit `/root`.